ixmp/backend/io.py

```diff
@@ -235,6 +260,11 @@ def s_read_excel(be, s, path, add_units=False, init_items=False,
         # Only parameters beyond this point
 
         df = xf.parse(name)
+        # appending data from repeated sheets due to max row limit
+        sheets = [x for x in xf.sheet_names if (name + '(') in x]
```

Comment:
Same as before + next if might be redundant

Reply:
Thanks, the startswith() used. But I believe the next if clause is needed because sheets maybe an empty list, then the following for loop raises an error. Was this your point?

Reply:
which error?

Reply:
You are right, I removed the if clause. thanks.